pkgs/tools/video/wtwitch/default.nix

```diff
@@ -0,0 +1,65 @@
+{ bash
+, coreutils-prefixed
+, curl
+, fetchFromGitHub
+, gnused
+, gnugrep
+, installShellFiles
+, jq
+, lib
+, makeWrapper
+, mplayer
+, mpv
+, ncurses
+, procps
+, scdoc
+, stdenv
+, streamlink
+, sudo
+, vlc
+}:
+stdenv.mkDerivation rec {
+  pname = "wtwitch";
+  version = "2.6.0";
+
+  src = fetchFromGitHub {
+    owner = "krathalan";
+    repo = pname;
+    rev = version;
+    hash = "sha256-KkuXZOquihY3IRVp4FM+AdN3kYi0MqmrXFuNmydTpio=";
+  };
+
+  buildPhase = ''
+    scdoc < src/wtwitch.1.scd > wtwitch.1
+  '';
+
+  nativeBuildInputs = [ scdoc installShellFiles makeWrapper ];
+
+  installPhase = ''
+    installManPage wtwitch.1
+    installShellCompletion --bash --name wtwitch.bash src/wtwitch-completion.bash
+    installShellCompletion --zsh --name _wtwitch src/_wtwitch
+    install -Dm755 src/wtwitch $out/bin/.wrapped/wtwitch
+    makeWrapper $out/bin/.wrapped/wtwitch $out/bin/wtwitch \
```

Comment:
```suggestion
    wrapProgram $out/bin/wtwitch \
```

Reply:
so, this was done to prevent the following, please advise:
```
$ wtwitch --help
The commands for wtwitch have changed.
Do not use hyphens anymore; simply execute ".wtwitch-wrapped help".

```

Reply:
The help message tells you to execute ``wtwitch help`` instead.

Reply:
correct, this is done so the help information correctly provides the binary name; this being a bash script they do a janky filename introspection, as opposed to using argv0, hence this is required.

Reply:
I would suggest to patch the program instead. Directories in $out/bin should not contain any subdirectories.

Reply:
sgtm; I have applied the same fix as #200157, see #150841